### DISCUSSION TITLE
Error since last update....

### DISCUSSION THREAD

**Discussion by sykick** [jertel/vuegraf#72] [Q&A] [answered]
Ever since the 1.3.3 update my docker container wont start. I've tried deleting a recreating the container but I get the same issue. Attached is the error log if it is any help. 

![vuegraf](https://user-images.githubusercontent.com/5907955/131662580-9e796cd1-5578-4e85-9f46-093588aa499f.jpg)

Any pointers on how to resolve would be good, was working fine until I updated....

rgds,
Marcus

**Comment by jertel** [comment, marked as answer]
Emporia changed their API again. I've pushed another fix up. Please pull `jertel/vuegraf:latest` and let me know if that gets you back online.

**Reply by sykick** [reply]
Prefect, just pulled the image again and it is all working. Thanks for the fast response, I very much appreciate your work on this project. 

rgds,
Marcus

**Reply by mcs123** [reply]
Awesome!, thank you @jertel

**Reply by dodgeboy** [reply]
I just started having this problem on 9/8/21 (after having had it previously, and upgraded to fix it).  I just pulled the repo again and it did not fix the problem this time.  How can I provide more info to help resolve this?  I really appreciate all of your effort on this.

**Reply by jertel** [reply]
Please start a new discussion since this one is already resolved. Also, see https://github.com/jertel/vuegraf/issues/73.
